garak/analyze/qual_review.py

```diff
@@ -0,0 +1,264 @@
+    "goodside.Tag": 1,
+    "goodside.ThreatenJSON": 2,
+    "goodside.WhoIsRiley": 2,
+    "grandma.Slurs": 1,
+    "grandma.Substances": 1,
+    "grandma.Win10": 2,
+    "grandma.Win11": 2,
+    "latentinjection.LatentInjectionFactSnippetEiffel": 1,
+    "latentinjection.LatentInjectionReport": 1,
+    "latentinjection.LatentInjectionResume": 1,
+    "latentinjection.LatentInjectionTranslationEnFr": 1,
+    "latentinjection.LatentInjectionTranslationEnZh": 1,
+    "latentinjection.LatentJailbreak": 1,
+    "latentinjection.LatentJailbreakMini": 1,
+    "leakreplay.GuardianCloze": 2,
+    "leakreplay.GuardianComplete": 1,
+    "leakreplay.LiteratureCloze": 2,
+    "leakreplay.LiteratureCloze80": 2,
+    "leakreplay.LiteratureComplete": 1,
+    "leakreplay.LiteratureComplete80": 1,
+    "leakreplay.NYTCloze": 2,
+    "leakreplay.NYTComplete": 1,
+    "leakreplay.PotterCloze": 2,
+    "leakreplay.PotterComplete": 1,
+    "lmrc.Anthropomorphisation": 3,
+    "lmrc.Bullying": 2,
+    "lmrc.Deadnaming": 2,
+    "lmrc.Profanity": 3,
+    "lmrc.QuackMedicine": 2,
+    "lmrc.SexualContent": 2,
+    "lmrc.Sexualisation": 2,
+    "lmrc.SlurUsage": 2,
+    "malwaregen.Evasion": 2,
+    "malwaregen.Payload": 2,
+    "malwaregen.SubFunctions": 2,
+    "malwaregen.TopLevel": 2,
+    "misleading.FalseAssertion50": 2,
+    "packagehallucination.JavaScript": 1,
+    "packagehallucination.PackageHallucinationProbe": "u",
+    "packagehallucination.Python": 1,
+    "packagehallucination.Ruby": 1,
+    "packagehallucination.Rust": 1,
+    "phrasing.FutureTense": 2,
+    "phrasing.FutureTenseMini": 2,
+    "phrasing.PastTense": 2,
+    "phrasing.PastTenseMini": 2,
+    "promptinject.HijackHateHumans": 2,
+    "promptinject.HijackHateHumansMini": 2,
+    "promptinject.HijackKillHumans": 2,
+    "promptinject.HijackKillHumansMini": 2,
+    "promptinject.HijackLongPrompt": 2,
+    "promptinject.HijackLongPromptMini": 2,
+    "realtoxicityprompts.RTPBlank": 1,
+    "realtoxicityprompts.RTPFlirtation": 3,
+    "realtoxicityprompts.RTPIdentity_Attack": 3,
+    "realtoxicityprompts.RTPInsult": 3,
+    "realtoxicityprompts.RTPProfanity": 3,
+    "realtoxicityprompts.RTPSevere_Toxicity": 3,
+    "realtoxicityprompts.RTPSexually_Explicit": 3,
+    "realtoxicityprompts.RTPThreat": 3,
+    "divergence.Repeat": 2,
+    "divergence.RepeatExtended": 2,
+    "snowball.GraphConnectivity": 2,
+    "snowball.GraphConnectivityMini": 2,
+    "snowball.Primes": 3,
+    "snowball.PrimesMini": 3,
+    "snowball.Senators": 3,
+    "snowball.SenatorsMini": 3,
+    "suffix.BEAST": 2,
+    "suffix.GCG": 2,
+    "suffix.GCGCached": 2,
+    "tap.PAIR": 2,
+    "tap.TAP": 2,
+    "tap.TAPCached": 2,
+    "test.Blank": "u",
+    "test.Test": "u",
+    "topic.WordnetAllowedWords": "u",
+    "topic.WordnetBlockedWords": "u",
+    "topic.WordnetControversial": 2,
+    "visual_jailbreak.FigStep": 2,
+    "visual_jailbreak.FigStepTiny": 2,
+    "xss.MarkdownImageExfil": 1,
+}
```

Comment:
See #1159 once that PR lands this should be removed and `tier` values can be obtained from the plugin cache via `_plugins.plugin_info()`.